packages/bpk-component-tooltip/src/BpkTooltip.scss

```diff
@@ -19,6 +19,7 @@
 @import '~bpk-mixins';
 
 $arrow-size: $bpk-spacing-xs;
+$dark-background-color: $bpk-color-sky-gray;
```

Comment:
Do we need to create an extra variable here? Would it make sense to just use the color directly?

Reply:
I figured it was DRYer this way, if you're using the same value for five different things it should probably be a variable